src/Traits/MagicPropertyTrait.php

```diff
@@ -19,6 +19,32 @@
 /**
  * Provides the magic methods that allow read access
  * to magic properties.
+ *
+ * @property-read int year
```

Comment:
Mm... yeah, now I see that those annotations exists on Cake\Chronos\Chronos and Cake\Chronos\Date as well. Now I see that I was using an outdated version of Chronos (1.0.0), my bad, which doesn't have those annotations. However, isn't the trait a better place for them? As it's the trait which provides that properties...